libs/model/src/models/reaction.ts

```diff
@@ -62,7 +65,15 @@ export default (
                 where: { id: thread_id },
               });
               if (thread) {
-                thread.increment('reaction_count');
+                await thread.increment('reaction_count', {
```

Comment:
I am going to approve this PR, but one more tiny nit.

We are making (and awaiting) 2 separate calls to increment/decrement the reaction_count and reaction_weights_sum, but these could be incremented in a single sql call since they are fields in the same table.

Also same argument can be made for comment increment/decrement below.

Reply:
Good suggestion. My local environment is a bit messed up right now so I don't want to further block this.

I've created a ticket to address that perf optimization: https://github.com/hicommonwealth/commonwealth/issues/6434